tests/test_render.py

```diff
@@ -777,3 +777,55 @@ def on_key_press(self, symbol: int, modifiers: int):
     # Make sure that the result is valid
     with open(IMAGE_FILENAME, "rb") as f:
         assert f.read() == png_snapshot
+
+
+@pytest.mark.required
```

Comment:
Do not mark this test as required, it is too long:
<img width="997" height="131" alt="image" src="https://github.com/user-attachments/assets/6789beca-a9d2-4f6f-91db-7ef661b679dd" />